### PR TITLE
New review process

### PR DESCRIPTION
**Pull request by bnmnetp**
Sorry for the noise... VSCode took the liberty of running prettier on the markdown file.

Just look at the section now called  "Rules for Merging PRs"

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,190 +1,198 @@
...
-* [Albert-Jan](https://github.com/albertjan)
-* [Meredydd Luff](https://github.com/meredydd)
-* [Leszek Swirski](https://github.com/LeszekSwirski)
-* [Ben Wheeler](https://github.com/bzwheeler)
+-   [Brad Miller](https://github.com/bnmnetp)
+-   [Scott Rixner](https://github.com/ixner)
+-   [Albert-Jan](https://github.com/albertjan)
+-   [Meredydd Luff](https://github.com/meredydd)
+-   [Leszek Swirski](https://github.com/LeszekSwirski)
+-   [Stu](https://github.com/s-cork)
```
Comment (s-cork) [inline CONTRIBUTING.md:181]:
```suggestion
-   [Stu Cork](https://github.com/s-cork)
```

**Review by s-cork — APPROVED** [review]
lgtm - just a couple of suggestions.

I'm curious how you'll change the github settings since at the moment PR requires one approval and you can't self approve.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bnmnetp) [reply CONTRIBUTING.md:181]: Thanks, I wasn't sure that Cork was your last name.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,190 +1,198 @@
...
+### Streamlined process for committers
+
+Committers can self approve a PR under the following conditions:
+
+-   The PR has thorough test coverage (and all tests pass)
+-   The PR has localized impact. For example it just affects one library module or one builtin...
+-   The PR is fixing a minor or cosmetic issue
+-   the PR is creating a relative improvement, for example adding a new module and removing a NotImplemented exception.
+
+If any of the above conditions are true, the committer should write up a quick justification as part of approving the PR and give it 48 hours in case anyone wants to object. After the 48 hour period then any committer, including the committer that created the PR can go ahead and merge.
```
[the remark this answers is not in the document]
Comment (bnmnetp) [reply CONTRIBUTING.md:159]: That is an excellent question.  I spent some time trying to figure out how to allow self approval, but that does not seem possible, so I simply removed the requirement for one review.  I'm open to suggestions.

**Comment by s-cork** [comment]
The only other way I can see is to make those who can merge without approval to be admins on the repo
There's a sneaky button for admins

Screenshot 2023-06-09 at 18 08 44

But also the current change is good too.

Merge when you're ready

**Pull request merged**